sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -213,6 +219,80 @@ public Long getSequenceNumber() {
         return systemProperties.getSequenceNumber();
     }
 
+    /**
+     * Gets the producer sequence number that was assigned during publishing, if the event was successfully
+     * published by a sequence-aware producer.  If the producer was not configured to apply
+     * sequence numbering or if the event has not yet been successfully published, this member
+     * will be {@code null}.
+     *
+     * The published sequence number is only populated and relevant when certain features
+     * of the producer are enabled. For example, it is used by idempotent publishing.
+     *
+     * @return The publishing sequence number assigned to the event at the time it was successfully published.
+     * {@code null} if the {@link EventData} hasn't been sent or it's sent without idempotent publishing enabled.
+     */
+    public Integer getPublishedSequenceNumber() {
+        return publishedSequenceNumber;
+    }
+
+    /**
+     * Gets the producer group id that was assigned during publishing, if the event was successfully
+     * published by a sequence-aware producer.  If the producer was not configured to apply
+     * sequence numbering or if the event has not yet been successfully published, this member
+     * will be {@code null}.
+     *
+     * The producer group id is only populated and relevant when certain features
+     * of the producer are enabled. For example, it is used by idempotent publishing.
+     *
+     * @return The producer group id assigned to the event at the time it was successfully published.
+     * {@code null} if the {@link EventData} hasn't been sent or it's sent without idempotent publishing enabled.
+     */
+    public Long getPublishedGroupId() {
+        return publishedGroupId;
+    }
+
+    /**
+     * Gets the producer owner level that was assigned during publishing, if the event was successfully
+     * published by a sequence-aware producer.  If the producer was not configured to apply
+     * sequence numbering or if the event has not yet been successfully published, this member
+     * will be {@code null}.
+     *
+     * The producer owner level is only populated and relevant when certain features
+     * of the producer are enabled. For example, it is used by idempotent publishing.
+     *
+     * @return The producer owner level assigned to the event at the time it was successfully published.
+     * {@code null} if the {@link EventData} hasn't been sent or it's sent without idempotent publishing enabled.
+     */
+    public Short getPublishedOwnerLevel() {
+        return publishedOwnerLevel;
+    }
+
+    void setProducerGroupIdInSysProperties(Long internalProducerGroupId) {
+        this.getSystemProperties().put(
+            AmqpMessageConstant.PRODUCER_ID_ANNOTATION_NAME.getValue(),
+            internalProducerGroupId
+        );
+    }
+
+    void setProducerOwnerLevelInSysProperties(Short internalProducerOwnerLevel) {
```

Comment:
I don't think we need "internal" as a parameter property name. Same with other ones above and below.

Reply:
Yeah, I changed the method names but not the param names 👎